docs/Installation.md

```diff
@@ -42,6 +38,25 @@ mkdir homebrew && curl -L https://github.com/Homebrew/brew/tarball/master | tar
 ### Multiple installations
 Create a Homebrew installation wherever you extract the tarball. Whichever `brew` command is called is where the packages will be installed. You can use this as you see fit, e.g. a system set of libs in `/usr/local` and tweaked formulae for development in `~/homebrew`.
 
+### OS X Mountain Lion (10.8) and below
```

Comment:
I reckon it's worth keeping this at the top to avoid issues being filed. Thoughts?

Reply:
Makes sense.